app.js

```diff
@@ -0,0 +1,190 @@
+  //     this.totalSales += this.dailySales[i];
+  //   }
+  //   var li = document.createElement('li');
+  //   li.innerText = 'Total Sales: ' + this.totalSales + ' cookies';
+  //   ul.appendChild(li);
+  // };
+
+  //function that creates a row of data for each store
+  this.createTableRow = function () {
+    var table = document.getElementById('store-data');
+    var tr = document.createElement('tr');
+    var tBody = document.getElementById('store-body');
+    var locationCell = document.createElement('td');
+    tBody.appendChild(tr);
+    //create table th
+    var th = document.createElement('th');
+    th.innerText = this.name;
+    tr.appendChild(th);
+    for (var i = 0; i < this.dailySales.length; i++) {
+      var td = document.createElement('td');
+      td.innerText = this.dailySales[i];
+      tr.appendChild(td);
+    }
+    // this.totalSalesPerLocation();
+    locationCell.innerText = this.locationTotals;
+    tr.appendChild(locationCell);
+  };
+
+   //function that calculates the total cookies for a given day
+  this.totalCookies = function () {
+    var tr = document.createElement('tr');
+    var tBody = document.getElementById('store-body');
+    tBody.appendChild(tr);
+    var totalSales = 0;
+    for (var i = 0; i < this.dailySales.length; i++) {
+      this.totalSales += this.dailySales[i];
+      var td = document.createElement('td');
+      td.innerText = this.totalSales;
+      tr.appendChild(td);
+    }
+  };
+
+  //this function renders that data to the page
+  this.renderSales = function() {
+    //calculate cookies
+    this.calcCookiesSales();
+    // creates the table
+    this.createTableRow();
+  };
+};
+
+//initializes the table for the page
+var createStoreTable = function () {
+  var main = document.getElementsByTagName('main')[0];
+  var table = document.createElement('table');
+  table.id = 'store-data';
+  main.appendChild(table);
+
+  //create table head
+  var tHead = document.createElement('thead');
+  table.appendChild(tHead);
+
+  var tBody = document.createElement('tbody');
+  tBody.id = 'store-body';
+  table.appendChild(tBody);
+
+  //create table tr
+  var tr = document.createElement('tr');
+  tHead.appendChild(tr);
+
+  //create table th
+  var th = document.createElement('th');
+  th.innerText = '';
+  tr.appendChild(th);
+  for (var i = 0; i < hours.length; i++) {
+    var th = document.createElement('th');
+    th.innerText = hours[i];
+    tr.appendChild(th);
+  }
+  var th2 = document.createElement('th');
+  th2.innerText = 'Location Totals: ';
+  tr.appendChild(th2);
+};
+
+// create table
+createStoreTable();
+
+//function calls
+var pike = new Store('1st & Pike',23,65,6.3);
+pike.renderSales();
+var airport = new Store('SeaTac Airport', 3,24,1.2);
+airport.renderSales();
+var center = new Store('Seattle Center', 11,38,3.7);
+center.renderSales();
+var hill = new Store('Capitol Hill', 20,38,2.3);
+hill.renderSales();
+var alki = new Store('Alki', 2, 16, 4.6);
+alki.renderSales();
```

Comment:
This is not points off, but you could probably group together all your constructor function calls first, then do all your `renderSales()` calls in a group after.